modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -2,162 +2,160 @@
+$ chcon -Rt svirt_sandbox_file_t .
 ----
+
+. In the same directory, run the installer:
 +
-For more information, see the _DockerHub, Cluster IP and Wildcard DNS Host values_ section in xref:additional-resources[Additional resources].
-+
-The following installation can take a while the first time it runs, as it pulls a number of Docker images. Once completed successfully, this results in an output similar to the following:
-+
+[source,bash]
 ----
-TASK [output-oc-cluster-status : debug] ******************************************************************************************************************************************************
-ok: [localhost] => {
-    "msg": [
-        "Web console URL: https://192.168.37.1:8443/console/",
-        "",
-        "Config is at host directory /var/lib/origin/openshift.local.config",
-        "Volumes are at host directory /var/lib/origin/openshift.local.volumes",
-        "Persistent volumes are at host directory /var/lib/origin/openshift.local.pv",
-        "Data is at host directory /path/to/mobile-core/ui/openshift-data"
-    ]
-}
-
-PLAY RECAP ***********************************************************************************************************************************************************************************
-localhost                  : ok=44   changed=17   unreachable=0    failed=0
+$ ./scripts/post_install.sh
 ----
 +
-The log above is displayed after a successful installation.
 
 . Verify the installation:
 +
-.. Browse to the Web console URL displayed at the end of the installation, and log in, accepting the self-signed certificate if displayed.
-+
-The developer login credentials are:
-+
-----
-username: developer
-password: password
-----
-+
-The service catalog is displayed
+.. Browse to the Web console of your OpenShift instance and log in.
 
-.. Check that the _Mobile_ tab is displayed in the service catalog. If this tab is not displayed, wait a few minutes to make sure that the installation process has completed. If the _Mobile_ tab still is not displayed, follow the troubleshooting steps below.
-
-image:catalog-mobile-clients.png[]
+.. Check that the _Mobile_ tab is displayed in the service catalog. If this tab is not displayed, wait a few minutes to make sure that the installation process has completed.
 
 [id='additional-resources']
 [discrete]
 = Additional resources
 
-== Troubleshooting
-If you have problems running {org-name} {product-name}, run the installer using the `--debug` option to capture as much information as possible:
-----
-$ ./installer/install.sh --debug
-----
+[[local-setup]]
+== Local OpenShift setup
 
-Firewall issues can occur with external devices trying to communicate with {product-name} provisioned on a Linux machine.
+You can spin up OpenShift locally on your machine. There are two scripts in Mobile Developer Console repository which will spin up the cluster using Minishift or `oc cluster up`, and enable {org-name} {product-name}.
```

Comment:
probably don't use `spin up`

Reply:
minishift is troublesome on linux, this suggests that it works on both Mac and Linux

Reply:
@pmacko1 what would be better to use instead of `spin up`?

Reply:
@finp thanks, fixed